contracts/EditionMetadataRenderer.sol

```diff
@@ -99,14 +100,14 @@ contract EditionMetadataRenderer is EditionMetadataState {
         return
             toBase64DataUrl(
                 string.concat(
-                    '{"name": "',
-                    name,
-                    '", "description": "',
-                    description,
+                    '{"name":"',
+                    LibString.escapeJSON(name),
+                    '","description":"',
+                    LibString.escapeJSON(description),
                     // this is for opensea since they don't respect ERC2981 right now
-                    '", "seller_fee_basis_points": ',
+                    '","seller_fee_basis_points":',
                     StringsUpgradeable.toString(royaltyBPS),
-                    ', "fee_recipient": "',
+                    ',"fee_recipient":"',
                     StringsUpgradeable.toHexString(royaltyRecipient),
```

Comment:
Can use `LibString.toHexString` instead.

Reply:
in [98d663a](https://github.com/showtime-xyz/nft-editions/pull/17/commits/98d663a712e3fdca82602b79cf5b0e35a22b3168)